app/src/main/res/values/strings.xml

```diff
@@ -25,12 +25,12 @@
     <string name="home_notice_content">Download Magisk ONLY from the official GitHub page. Files from unknown sources can be malicious!</string>
     <string name="home_support_title">Support Us</string>
     <string name="home_item_source">Source</string>
-    <string name="home_support_content">Magisk is, and always will be, free and open-source. You can however show us that you care by sending a small donation.</string>
+    <string name="home_support_content">Magisk is, and always will be, free, and open source. You can however show us that you care by making a donation.</string>
     <string name="home_installed_version">Installed</string>
     <string name="home_latest_version">Latest</string>
     <string name="invalid_update_channel">Invalid Update Channel</string>
     <string name="uninstall_magisk_title">Uninstall Magisk</string>
-    <string name="uninstall_magisk_msg">All modules will be disabled/removed!\nRoot will be removed!\nYour data potentially encrypted if not already!</string>
+    <string name="uninstall_magisk_msg">Root will be removed!\nAll modules will be removed!\nAny internal storage decrypted through the use of Magisk will be re-encrypted!</string>
```

Comment:
Restore

Reply:
They wouldn't be disabled in that case, just dormant.

Reply:
@vvb2060 If you can put in a full suggestion it makes it easier to decide for or against.
"All modules will be turned off" seems to cover both disabled modules and removed ones, though it sounds strange.
How far towards removal does disabling modules go, and is it the case for all of them?

Reply:
Are you a magisk user, or just a translator? Have you checked how the strings look like when displayed on the app UI? If user has chosen to restore the boot image, then because there is no magisk, of course no module will be loaded, but the module itself is not removed.

Reply:
@vvb2060 I checked how they looked beforehand. This looks to be a bit longer.
I understand that, but I can't and am not trying to assume your knowledge beyond that.
If it is "disabled and removed", the "disabled" part is redundant to the end result.
If it is "removed or disabled" I don't know specifically what modules that applies to, or what "disabled" really entails.
What gets left behind short of "removed"?

"All modules will be unloaded" covers all, but isn't precise.

Reply:
Right, so it depends on the option chosen. I guess the original "disabled/removed" is more accurate since it applies to two very different scenarios.

Reply:
Corrected. 👍